src/main/java/org/cbio/gdcpipeline/listener/MutationStepListener.java

```diff
@@ -87,7 +88,20 @@ public ExitStatus afterStep(StepExecution stepExecution) {
     }
 
     public List<File> getMutationFileList() {
-        return CommonDataUtil.getFileList(gdcFileMetadatas, CommonDataUtil.GDC_TYPE.MUTATION, sourceDir);
+        List<File> mutationFileList = new ArrayList<>();
```

Comment:
The code looks good here. But just comparing to what the previous code here did, it looks like we're losing explicit checks on path existence/differentiating between compressed/uncompressed files. I'm not as familiar with what the data looks like when we work with it, is this okay to drop?

Reply:
I think I should have just refactored that method in the util class. I just wrote some code that does exactly what you just described